modules/elb.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+from helper import get_all_instances, get_all_rds, get_all_sg, get_all_elbs
+import crayons
+import boto3
+import config
+
+
+def scan():
+    not_used = []
+    flags = []
+
+    response = get_all_elbs()
+    for ELB in response['LoadBalancerDescriptions']:
+        if len(ELB['Instances']) == 0:
```

Comment:
Stuff like this should already be done in the helper function. `get_all_elbs` should just return a list of all ELBs where you don't need to do any more checks or processing of the data